torch/jit/_shape_functions.py

```diff
@@ -922,6 +922,33 @@ def native_batch_norm(input: List[int], weight: Optional[List[int]], bias: Optio
         _size = [0]
     return _copy(input), _size, _size
 
+def split_fn(input: List[int], split_sizes: List[int], dim: int = 0):
+    dim = maybe_wrap_dim(dim, len(input))
+    num_indices = len(split_sizes)
+    results: List[List[int]] = []
+    start = 0
+    for i in range(num_indices):
+        end = split_sizes[i]
+        results[i] = slice(input, dim, start, end, step = 1)
+        start = end
+    return results
```

Comment:
This is breaking due to the outstanding bug that should be fixed if  #77120 is landed
@eellison , are you planning to land the related change?

Reply:
Hey, there were failures which I havnen't looked into, do you mind picking it up?

Reply:
Yeah, I can pick it up.